apps/web/package.json

```diff
@@ -27,6 +27,7 @@
     "@azure/msal-node": "1.8.0",
     "@hapi/boom": "9.1.4",
     "accessible-autocomplete": "2.0.4",
+    "babel-jest": "^27.5.1",
```

Comment:
Do we need this though? Is it for using ESM within the test files?

Reply:
Yes, for exactly that.